src/messages/generic_api.py

```diff
@@ -42,4 +44,7 @@ def _send(self, msg: str):
 
         data = json.loads(self._config.advanced_options['message_body_template'].format(msg=msg), strict=False)
 
-        requests.post(url, headers=headers, json=data)
+        try:
+            requests.post(url, headers=headers, json=data)
+        except BaseException as err:
+            Logger.error("Error sending generic Discord message: " + err)
```

Comment:
Not necessarily a discord message, update logger message to some like generic message? Little nit-picky I know

Reply:
@egut125 What do you mean exactly?

Reply:
Remove the word discord from the string. Since it could be used for something other than discord.

Reply:
@egut125 Ah, gotcha. Done.